frontend/src/pages/journal/components/journal-sidebar/journal-sidebar.tsx

```diff
@@ -53,17 +71,30 @@ const JournalSidebar: React.FC<Properties> = ({
         <div className={styles['info']}>
           <span>Journal</span>
         </div>
+        <Button
+          label="Add note"
+          isLabelVisuallyHidden
+          iconName="plus"
+          style="add"
+          onClick={handlePlusButtonClick}
+        />
       </SidebarHeader>
       <SidebarBody>
         <div className={styles['journal-entry-list']}>
           {allJournalEntries.map((journalEntry) => {
+            const noteLink = (AppRoute.JOURNAL_ENTRY_$ID.replace(
+              ':id',
+              String(journalEntry.id),
+            ) + '?sidebarMode=hide') as typeof AppRoute.JOURNAL_ENTRY_$ID;
```

Comment:
it's useless here

Reply:
If you want to hide a sidebar prefer using callback for it

Reply:
I did it like it is in the chatSidebar component, which was merged. Also, there is a [https://github.com/orgs/BinaryStudioAcademy/projects/12/views/1?sliceBy%5Bvalue%5D=HIMDanny&pane=issue&itemId=38741468](bug) created for this. So it is a temporary solution.